.gitignore

```diff
@@ -12,5 +12,5 @@ env
 *.pyc
 *.code-workspace
 tmp
-data
+/data
```

Comment:
Is that right? Doesn't that actually mean the root? Anything outside the git repo shouldn't be affected by gitignore anyway no?